src/Workspaces/Core/Portable/CodeActions/CodeAction.cs

```diff
@@ -452,7 +452,7 @@ internal class SolutionChangeAction : SimpleCodeAction
             public SolutionChangeAction(
                 string title,
                 Func<CancellationToken, Task<Solution>> createChangedSolution,
-                string? equivalenceKey = null,
+                string? equivalenceKey,
```

Comment:
I didn't change this to non-nullable because it's called by a public API:

https://github.com/dotnet/roslyn/blob/1a4922e21af849b3368c9cb41d865e91643c83ac/src/Workspaces/Core/Portable/CodeActions/CodeAction.cs#L335

I'll have to either add a suppression or change the public API nullability. I thought it's good enough for this to not be optional.